### PR TITLE
get-substrate.sh should set default toolchain when installing rust

### PR DESCRIPTION
**Pull request by gabreal**

### CODE REVIEW

File: get-substrate.sh [hunk 1]
```diff
@@ -52,6 +52,7 @@ fi
 if ! which rustup >/dev/null 2>&1; then
 	curl https://sh.rustup.rs -sSf | sh -s -- -y
 	source ~/.cargo/env
+  rustup default stable
```
Comment (TriplEight) [inline get-substrate.sh]: Indent please

File: get-substrate.sh [hunk 2]
```diff
@@ -52,6 +52,7 @@ fi
 if ! which rustup >/dev/null 2>&1; then
 	curl https://sh.rustup.rs -sSf | sh -s -- -y
 	source ~/.cargo/env
+	srustup default stable
```
Comment (TriplEight) [inline get-substrate.sh]: typo ^^

**Comment by bkchr** [comment]
Hmm, I really don't like to force people into what we think is correct. If we need stable, then we need to run `cargo +stable build`, but otherwise I would like to not alter systems by users.

**Comment by bkchr** [comment]
Okay, it is just for installing. Forget what I said.

**Pull request merged**